selfdrive/ui/translations/main_th.ts

```diff
@@ -0,0 +1,1274 @@
+        <translation>ยกเลิกระบบช่วยขับเพื่อปิดเครื่อง</translation>
+    </message>
+</context>
+<context>
+    <name>DriveStats</name>
+    <message>
+        <location filename="../qt/widgets/drive_stats.cc" line="37"/>
+        <source>Drives</source>
+        <translation>การขับขี่</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/drive_stats.cc" line="39"/>
+        <source>Hours</source>
+        <translation>ชั่วโมง</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/drive_stats.cc" line="44"/>
+        <source>ALL TIME</source>
+        <translation>ทั้งหมด</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/drive_stats.cc" line="46"/>
+        <source>PAST WEEK</source>
+        <translation>สัปดาห์ที่ผ่านมา</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/drive_stats.h" line="15"/>
+        <source>KM</source>
+        <translation>ก.ม.</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/drive_stats.h" line="15"/>
+        <source>Miles</source>
+        <translation>ไมล์</translation>
+    </message>
+</context>
+<context>
+    <name>DriverViewScene</name>
+    <message>
+        <location filename="../qt/offroad/driverview.cc" line="56"/>
+        <source>camera starting</source>
+        <translation>กำลังเปิดกล้อง</translation>
+    </message>
+</context>
+<context>
+    <name>InputDialog</name>
+    <message>
+        <location filename="../qt/widgets/input.cc" line="71"/>
+        <source>Cancel</source>
+        <translation>ยกเลิก</translation>
+    </message>
+    <message>
+        <location filename="../qt/widgets/input.cc" line="168"/>
+        <source>Need at least %1 characters!</source>
+        <translation>ต้องการอย่างน้อย %1 ตัวอักษร!</translation>
+    </message>
+</context>
+<context>
+    <name>Installer</name>
+    <message>
+        <location filename="../installer/installer.cc" line="56"/>
+        <source>Installing...</source>
+        <translation>กำลังติดตั้ง...</translation>
+    </message>
+    <message>
+        <location filename="../installer/installer.cc" line="144"/>
+        <source>Receiving objects: </source>
+        <translation>กำลังรับข้อมูล: </translation>
+    </message>
+    <message>
+        <location filename="../installer/installer.cc" line="145"/>
+        <source>Resolving deltas: </source>
+        <translation>การแก้ไขเดลต้า: </translation>
+    </message>
+    <message>
+        <location filename="../installer/installer.cc" line="146"/>
+        <source>Updating files: </source>
+        <translation>กำลังอัปเดตไฟล์: </translation>
+    </message>
+</context>
+<context>
+    <name>MapETA</name>
+    <message>
+        <location filename="../qt/maps/map.cc" line="622"/>
+        <source>eta</source>
+        <translation>เวลาถึงปลายทางโดยประมาณ</translation>
```

Comment:
Unfortunately this needs to be quite short to fit in the provided space with the current design. Shall we use `eta` instead?

Reply:
Yes, we can use "eta" instead. No Thai word to replace this. I will update it.